README.md

```diff
@@ -23,7 +23,8 @@ These instructions will get you a copy of the role for your ansible playbook. On
 
 ### Prerequisities
 
-Ansible 2.2.1.0 version installed.
+Ansible 2.4.2.0 version installed.
```

Comment:
2.4.2.0 or 2.4.0.0? In `travis.yml` and `meta/main.yml` is 2.4.0.0

Reply:
fixed